socket-ballerina/build.gradle

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import org.apache.tools.ant.taskdefs.condition.Os
+
+apply from: "$rootDir/gradle/mvnPublisher.gradle"
+
+description = 'Ballerina - Socket Ballerina Generator'
+
+configurations {
+    balTools
+    socketBallerinaBirDir
+    socketBallerinaJar
+}
+
+dependencies {
+    balTools ("org.ballerinalang:jballerina-tools:${ballerinaLangVersion}") {
+        transitive = false
+    }
+    compile project(':socket-native')
+}
+
```

Comment:
Let's remove these additional empty lines.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-socket/pull/2/commits/99fccd29af679a7560130308c6579f021f53c1cf